src/Footer.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+function Footer(props) {
+  return (
+    <div>
+      <ul>
+        {props.contact.map(string => {
```

Comment:
do you see any warnings in the console about a missing key? Can you remember how to fix that (hint: it's in the syllabus docs from lesson 1)